features/admin.identity-providers.v1/meta/authenticator-meta.ts

```diff
@@ -159,6 +159,17 @@ export class AuthenticatorMeta {
         return icon ?? getAuthenticatorIcons().default;
     }
 
+    /**
+     * Get Custom Authenticator Icon.
+     * Currently authenticator id is being used to fetch the respective authenticator icon.
+     * Existing function could not be used since the id and the name of
+     * custom authenticators are not pre defined.
```

Comment:
```suggestion
     *
     * Currently authenticator id is being used to fetch the respective authenticator icon.
     * Existing function could not be used since the id and the name of
     * custom authenticators are not pre defined.
     *
```